openreview/venue_request/process/revisionProcess.py

```diff
@@ -214,6 +214,12 @@ def process(client, note, invitation):
                 #update post submission hide_fields
                 submission_content = conference.submission_stage.get_content(api_version='2', conference=conference)
                 hide_fields = [key for key in submission_content.keys() if key not in ['title', 'authors', 'authorids', 'venue', 'venueid'] and 'delete' not in submission_content[key]]
+                second_deadline_fields = forum_note.content.get('second_deadline_additional_options', {})
+                if second_deadline_fields:
+                    fields = list(second_deadline_fields.keys())
+                    for field in fields:
+                        if field not in hide_fields and 'delete' not in second_deadline_fields[field]:
+                            hide_fields.append(field)
```

Comment:
do you need to do this because there is no way to set the hide_fields in the post submission button?

I'm worried that we are building the hide_fields on the fly and it is not persisted in the request note.

Reply:
I do this because if they add fields to the submissions using the `second_deadline_fields` field, then these fields will not appear in the `hide_fields` dropdown and PCs won't be able to hide them. The alternative is to have `hide_fields` be a textbox instead of a dropdown, and allow PCs to add any string there.

Reply:
is there a chance to build the dropdown values reading the fields from `additional_submission_options` and `second_deadline_fields`?

Reply:
it seems we are building the dropdown using the submission content:

https://github.com/openreview/openreview-py/blob/master/openreview/venue_request/process/revisionProcess.py#L216

I think we should include the `second_deadline_fields` or get the revision content?

Reply:
Well I am adding the `second_deadline_fields` here too, or is this not what you mean?

Reply:
ohhhh I see, we are talking about the same thing! sorry about that, I thought this code was creating the revision invitation 🤦‍♀️  sorry I'm too 🤰